src/io/utilities/parseSBMLNotesField2012.m

```diff
@@ -63,15 +63,15 @@
     elseif (regexp(fieldStr,'^FORMULA'))
         formula = regexprep(strrep(fieldStr,'FORMULA:',''),'^(\s)+','');
     elseif (regexp(fieldStr,'^CHARGE'))
-        charge = str2num(regexprep(strrep(fieldStr,'CHARGE:',''),'^(\s)+',''));
+        charge = str2double(regexprep(strrep(fieldStr,'CHARGE:',''),'^(\s)+',''));
     elseif (regexp(fieldStr,'AUTHORS'))
         if isempty(citation)
             citation = strcat(regexprep(strrep(fieldStr,'AUTHORS:',''),'^(\s)+',''));
         else
             citation = strcat(citation,';',regexprep(strrep(fieldStr,'AUTHORS:',''),'^(\s)+',''));
         end
     elseif (regexp(fieldStr,'^Confidence Level'))
-        confidenceScore = regexprep(strrep(fieldStr,'Confidence Level:',''),'^(\s)+','');
+        [matches, tmpTokens] = regexpi(fieldStr, 'Confidence[ _]Level: (\w+)', 'match', 'tokens');
```

Comment:
The comment is correct, but...
Is the confidenceScore a String value or a Double value? Up until now, it was present as a double in the models. 
The toolbox currently assumes, that this is a cell array (see e.g. addReaction, where a new reaction appends to the confidence score as a cell array). 
So for now, I would add :
if (~isempty(matches))
confidenceScore = tmpTokens{1}{1};
end
instead of the str2double version. But this should also go into our discussion about model fields (and again this is almost unused in any other toolbox function, so changing it should not be a big issue).